src/client/eppo-client.ts

```diff
@@ -866,12 +866,14 @@ export default class EppoClient {
    *
    * @param subjectKey an identifier of the experiment subject, for example a user ID.
    * @param subjectAttributes optional attributes associated with the subject, for example name and email.
-   * @param banditActions
+   * @param banditActions optional attributes associated with the bandit actions
+   * @param salt a salt to use for obfuscation
    */
   getPrecomputedConfiguration(
     subjectKey: string,
     subjectAttributes: Attributes | ContextAttributes = {},
     banditActions: Record<FlagKey, BanditActions> = {},
+    salt?: string,
```

Comment:
```suggestion
    salt: string | undefined = '',
```

Better yet - can the salt be required?

Reply:
It cannot easily. 

Unlike the js client sdk, the node server sdk doesn't extend the client class from common, but instead directly exposes each of these methods to the end user. In other words, requiring a salt here would mean that calling `getPrecomputedConfiguration` in the node sdk would also require the user to provide a salt. But this requirement of the user is unnecessary because the sdk can generate the salt. 

I think my approach here is okay. If the user provides a salt, it will override the random salt generator, otherwise, we generate it randomly in the node sdk.